### ISSUE TITLE
Cannot start local mock backend

### ISSUE THREAD

**Issue by zxul767** [wizeline/wizelink-front#18]
Running `> $ npm run start-api` as the instructions say, leads to:

```
> wizelink-front@0.1.0 start-api /Users/zxul767/src/wizeline/side-projects/wizelink-front
> nodemon src/utils/server.js

sh: nodemon: command not found

npm ERR! Darwin 16.6.0
npm ERR! argv "/usr/local/Cellar/node/7.8.0/bin/node" "/usr/local/bin/npm" "run" "start-api"
npm ERR! node v7.8.0
npm ERR! npm  v4.2.0
npm ERR! file sh
npm ERR! code ELIFECYCLE
npm ERR! errno ENOENT
npm ERR! syscall spawn
npm ERR! wizelink-front@0.1.0 start-api: `nodemon src/utils/server.js`
npm ERR! spawn ENOENT
npm ERR!
npm ERR! Failed at the wizelink-front@0.1.0 start-api script 'nodemon src/utils/server.js'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the wizelink-front package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     nodemon src/utils/server.js
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs wizelink-front
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls wizelink-front
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/zxul767/.npm/_logs/2017-07-25T01_15_34_229Z-debug.log
```

**Comment by vusause** [comment]
Sorry, I forgot to include that nodemon must be installed either globally or locally within the project
`npm install -g nodemon` should fix this

**Comment by elsypinzonv** [comment]
@vusause If there aren't any problem, I going to create a branch to fix it. ;)

**Comment by zxul767** [comment]
Fixed by #20
